wiki/Rules/Contributor_Code_of_Conduct/de.md

```diff
@@ -0,0 +1,84 @@
+
+- [peppy](https://osu.ppy.sh/users/2) ([pe@ppy.sh](mailto:pe@ppy.sh))
+- [Ephemeral](https://osu.ppy.sh/users/102335) ([ephemeral@ppy.sh](mailto:ephemeral@ppy.sh))
+
+Alle Beschwerden werden unverzüglich und fair überprüft sowie untersucht.
+
+Alle Community-Leiter sind dazu verpflichtet, die Privatsphäre und Sicherheit des Berichterstatters jedes Vorfalls zu respektieren und zu wahren.
+
+## Richtlinien für die Durchsetzung
+
+Die Führungspersönlichkeiten der Community werden sich an diese Richtlinien für die Auswirkungen auf die Community richten, um die Konsequenzen jedes Handelns zu bestimmen, das sie als Verstoß gegen diesen Verhaltenskodex ansehen:
+
+### 1. Berichtigung
+
+**Auswirkungen auf die Community**: Verwendung einer unangemessenen Ausdrucksweise oder eines anderen Verhaltens, das in der Community als unprofessionell oder unerwünscht gilt.
+
+**Konsequenz**: Eine privat adressierte Warnung von den Community-Leitern, die Klarheit über die Art des Verstoßes schafft und eine Erklärung bietet, warum dieses Verhalten unangemessen war. Eine öffentliche Entschuldigung kann verlangt werden.
+
+### 2. Warnung
+
+**Auswirkungen auf die Community**: Ein Verstoß durch einen einzigen Vorfall oder eine Reihe an Aktionen.
+
+**Konsequenz**: Eine Warnung mit Konsequenzen für anhaltendes Verhalten. Keine Interaktion für einen bestimmten Zeitraum mit den involvierten Personen, einschließlich unaufgeforderte Interaktionen mit denen, die den Verhaltenskodex durchsetzen. Das beinhaltet die Vermeidung von Interaktionen an Community-Orten sowie in externen Kanälen wie Social-Media. Ein Verstoß gegen diese Bedingungen kann zu einer vorübergehenden oder dauerhaften Sperre führen.
+
+### 3. Temporäre Sperre
+
+**Auswirkungen auf die Community**: Ein schwerwiegender Verstoß gegen die Community-Standards, einschließlich anhaltend unangemessenes Verhalten.
+
+**Konsequenz**: Eine vorübergehende Sperre von jeder Art von Interaktion oder öffentlicher Kommunikation mit der Community für einen bestimmten Zeitraum. Keine öffentlichen oder privaten Interaktionen mit den involvierten Personen ist während dieser Zeitspanne erlaubt, einschließlich unaufgeforderte Interaktionen mit denen, die den Verhaltenskodex durchsetzen. Ein Verstoß gegen diese Bedingungen kann zu einer permanenten Sperre führen.
+
+### 4. Dauerhafte Sperre
+
+**Auswirkungen auf die Community**: Das Vormachen von Verstößen gegen die Community-Standards, einschließlich anhaltend unangemessenes Verhalten, Belästigung von Einzelpersonen oder Aggression oder Herabwürdigung gegen Personengruppen.
+
+**Konsequenz**: Eine dauerhafte Sperre von jeder Art von öffentlicher Interaktion innerhalb der Community.
+
+## Quellen
+
+Dieser Verhaltenskodex wurde aus dem [Contributor Covenant](https://www.contributor-covenant.org) adaptiert, verfügbar unter <https://www.contributor-covenant.org/version/2/0/code_of_conduct.html>.
```

Comment:
```suggestion
Dieser Verhaltenskodex wurde aus der [Vereinbarung über den Verhaltenskodex für Mitwirkende](https://www.contributor-covenant.org) adaptiert, verfügbar unter <https://www.contributor-covenant.org/de/version/2/0/code_of_conduct>.
```

Might as well link the official translation imo
https://www.contributor-covenant.org/de/version/2/0/code_of_conduct